daemons/dnssec/ipa-ods-exporter

```diff
@@ -516,7 +521,9 @@ def parse_command(cmd):
 def send_systemd_reply(conn, reply):
         # Reply & close connection early.
         # This is necessary to let Enforcer to unlock the ODS DB.
-        conn.send(reply + '\n')
+        if isinstance(reply, six.text_type):
```

Comment:
I would personally rather set an expectation for the function to receive string (unicode in Python2) and then encode it right away. I've been recently playing with Haskell and its static typing had a great influence on me 😛 I believe MyPy (should we ever include its checks for FreeIPA) would also appreciate that.